data/mods/Magiclysm/enchanted/enchanted_wands.json

```diff
@@ -0,0 +1,362 @@
+[
+  {
+    "id": "wand_magic_missile_minor",
+    "name": "wand of magic missiles",
+    "plural": "wands of magic missile",
+    "description": "A classic, this wand shoots magic missiles without fail.  A more skilled Magus could cast them more efficiently, though.",
+    "type": "TOOL",
+    "weight": 230,
+    "volume": 2,
+    "price": 10000,
+    "bashing": 3,
+    "to_hit": 2,
+    "material": [ "wood" ],
+    "symbol": "|",
+    "color": "brown",
+    "charges_per_use": 1,
+    "ammo": "crystallized_mana",
+    "use_action": { "type": "cast_spell", "spell_id": "magic_missile", "no_fail": true, "level": 5 },
+    "magazines": [ [ "crystallized_mana", [ "small_mana_crystal" ] ] ],
+    "magazine_well": 1
+  },
+  {
+    "id": "wand_magic_missile_lesser",
+    "name": "lesser wand of magic missiles",
+    "plural": "wands of magic missile",
+    "description": "A classic, this wand shoots magic missiles without fail.  A more skilled Magus could cast them more efficiently, though.",
+    "type": "TOOL",
+    "weight": 230,
+    "volume": 2,
+    "price": 10000,
+    "bashing": 3,
+    "to_hit": 2,
+    "material": [ "wood" ],
+    "symbol": "|",
+    "color": "brown",
+    "charges_per_use": 1,
+    "ammo": "crystallized_mana",
+    "use_action": { "type": "cast_spell", "spell_id": "magic_missile", "no_fail": true, "level": 10 },
+    "magazines": [ [ "crystallized_mana", [ "small_mana_crystal" ] ] ],
+    "magazine_well": 1
+  },
+  {
+    "id": "wand_magic_missile_greater",
+    "name": "greater wand of magic missiles",
+    "plural": "wands of magic missile",
+    "description": "A classic, this wand shoots magic missiles without fail.  A more skilled Magus could cast them more efficiently, though.",
+    "type": "TOOL",
+    "weight": 230,
+    "volume": 2,
+    "price": 10000,
+    "bashing": 3,
+    "to_hit": 2,
+    "material": [ "wood" ],
+    "symbol": "|",
+    "color": "brown",
+    "charges_per_use": 1,
+    "ammo": "crystallized_mana",
+    "use_action": { "type": "cast_spell", "spell_id": "magic_missile", "no_fail": true, "level": 15 },
+    "magazines": [ [ "crystallized_mana", [ "small_mana_crystal" ] ] ],
+    "magazine_well": 1
+  },
+  {
+    "id": "wand_fireball_minor",
```

Comment:
same as above but fireball

Reply:
No, I intended these to be replacements for those as they don't have any descriptive determination of their power level, those should be moved into a file for obsolete items and these spawn from now on.

Reply:
very well, please do so in this pr

Reply:
> 
> 
> You've got some copy paste errors in the magic wands

Are you referring to the descriptive text? I mentioned it needed to be done on discord, it'll be taken care of. Or did I miss something else?